massa-bootstrap/src/listener.rs

```diff
@@ -48,26 +46,29 @@ impl BootstrapTcpListener {
             socket.set_only_v6(false)?;
         }
         // This is needed for the mio-polling system, which depends on the socket being non-blocking.
-        // If we don't set non-blocking, then we can .accept() on the mio_server bellow, which is needed to ensure the polling triggers every time.
+        // If we don't set non-blocking, then we can .accept() on the server below, which is needed to ensure the polling triggers every time.
         socket.set_nonblocking(true)?;
         socket.bind(&(*addr).into())?;
 
         // Number of connections to queue, set to the hardcoded value used by tokio
         socket.listen(1024)?;
 
         info!("Starting bootstrap listener on {}", &addr);
-        let server: TcpListener = socket.into();
+        let std_server: std::net::TcpListener = socket.into();
 
-        let mut mio_server =
-            MioTcpListener::from_std(server.try_clone().expect("Unable to clone server socket"));
+        let mut server = TcpListener::from_std(
```

Comment:
instead of `let mut server = TcpListener::from_std(std_server.try_clone().expect("Unable to clone server socket"));` why not just `let mut server = TcpListener::from_std(std_server);` ?

I want to avoid keeping useless references from different libs to the same object at the same time

Reply:
Yes, the clone is no longer needed as we don't keep the std_server! Thanks for pointing it out